data/mur_citation_map.csv

```diff
@@ -0,0 +1,172 @@
+2:431,52:30101
```

Comment:
Ah, thanks for catching this folks. I was using the complete title 52 table, but it sounds like that's unnecessary and the original `reclassify_statutory_citation` method would've worked fine here. I'll simplify this once more.